### PR TITLE
480 append test to module soul-sync-data-http

### PR DESCRIPTION
**Pull request by 0x12FD16B**
Make sure that:

- [x] You have read the [contribution guidelines](https://dromara.org/en-us/docs/soul/contributor.html).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

**Comment by yu199195** [comment]
把master上最的代码合并到你的分支，本地先install跑过

**Comment by tydhot** [comment]
本地先install一下 不然checkstyle过不了

**Comment by 0x12FD16B** [comment]
@huangxfchn please review

File: soul-sync-data-center/soul-sync-data-http/src/test/java/org/dromara/soul/sync/data/http/support/MockHttpDataSyncEndpoint.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.server.handler.ContextHandler;
+import org.eclipse.jetty.server.handler.ContextHandlerCollection;
+
+/**
+ * mock Http Data Sync Endpoint.
+ *
+ * @author David Liu
+ */
+public class MockHttpDataSyncEndpoint {
```
Comment (dengliming) [inline soul-sync-data-center/soul-sync-data-http/src/test/java/org/dromara/soul/sync/data/http/support/MockHttpDataSyncEndpoint.java:29]: 这个其实可以不用jetty，可以考虑下能否参考这里[https://github.com/dromara/soul/blob/master/soul-plugin/soul-plugin-httpclient/src/test/java/org/dromara/soul/plugin/httpclient/WebclientTest.java#L42](https://github.com/dromara/soul/blob/master/soul-plugin/soul-plugin-httpclient/src/test/java/org/dromara/soul/plugin/httpclient/WebclientTest.java#L42)

Comment (0x12FD16B) [reply soul-sync-data-center/soul-sync-data-http/src/test/java/org/dromara/soul/sync/data/http/support/MockHttpDataSyncEndpoint.java]: @dengliming 换成 WireMock 了, 之前使用 Jetty 的原因是: Mock 多个 Http 接口, 代码组织结构会更清晰。
运行完成之后发现只需要 Mock 两个接口

**Review by huangxfchn — APPROVED** [review]
good job.

**Pull request merged**